gradle.properties

```diff
@@ -1,3 +1,21 @@
+GROUP=app.cash.duktape
+VERSION_NAME=1.4.0-SNAPSHOT
+
+POM_DESCRIPTION=The Duktape embeddable JavaScript engine packaged for Android.
+
+POM_URL=https://github.com/cashapp/duktape-android/
+POM_SCM_URL=https://github.com/cashapp/duktape-android/
+POM_SCM_CONNECTION=scm:git:git://github.com/cashapp/duktape-android.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/cashapp/duktape-android.git
+
+POM_LICENCE_NAME=Apache-2.0
+POM_LICENCE_URL=https://www.apache.org/licenses/LICENSE-2.0
+POM_LICENCE_DIST=repo
+
+POM_DEVELOPER_ID=cashapp
```

Comment:
All this has changed, too. Intentional?

Reply:
Yeah I mean the repo moved and this is what we use on other cashapp projects. The values are mostly irrelevant but are required.